styleguide/derek/global-components/ceiling/ceilingDark/_ceilingDark.scss

```diff
@@ -0,0 +1,69 @@
+.ceiling-menuInline-item {
+  border-left: 1px solid $white;
+}
```

Comment:
This should use a selector specific to the dark ceiling.  It's coloring the border for the standard landing pages. 

![image](https://cloud.githubusercontent.com/assets/5263371/26453284/d7a9b95e-4128-11e7-9d5b-de6c768dd418.png)